tests/Plugin_Main_Tests.php

```diff
@@ -12,7 +12,7 @@ class Plugin_Main_Tests extends WP_UnitTestCase {
 	public function set_up() {
 		parent::set_up();
 
-		$this->plugin_main = new Plugin_Main( basename( dirname( __DIR__ ) ) . '/plugin-check.php' );
+		$this->plugin_main = new Plugin_Main( plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE ) );
```

Comment:
While related to my suggestions above, this code here is actually incorrect (also was incorrect before): It needs to be the full path to the plugin main file, not the plugin base name (see parameter documentation).
```suggestion
		$this->plugin_main = new Plugin_Main( WP_PLUGIN_CHECK_MAIN_FILE );
```